exercises/two-bucket/example.py

```diff
@@ -1,14 +1,13 @@
 '''
-    This solution implements a breadth-first search of the graph
+    This solution implements a breadth-start_bucket search of the graph
```

Comment:
This appears to be an unintentional change from a Replace-All; please revert it.